templateutils/templateutils.go

```diff
@@ -63,8 +64,27 @@ Some new functions have been added to Go's template language
   field for each struct on a new line , e.g.,
 
   {{select . "Name"}}
+- table outputs a table given an array or a slice of structs.  The table headings are taken
+  from the names of the structs fields.  Hidden fields and fields of type channel are ignored.
+  The tabwidth and minimum column width are hardcoded to 8.  An example of table's usage is
+
+  {{table .}}
+- tablex is similar to table but it allows the caller more control over the table's appearance.
```

Comment:
I'm blanking on something more descriptive, but the name bugs me.  I'm not in love with "tablef" either, but that's all that comes to mind.